src/features/surveys/l10n/messageIds.ts

```diff
@@ -179,6 +179,16 @@ export default makeMessages('feat.surveys', {
     ),
     title: m('Add email address'),
   },
+  surveyFooter: {
+    links: {
+      foundation: m('Zetkin Foundation'),
+      manual: m('Zetkin Manual'),
```

Comment:
This can be removed.

Reply:
✅ https://github.com/zetkin/app.zetkin.org/pull/2447/commits/77fec5bc133191e4b088510d85f13e6f4ea2301c